frontend/public/components/edit-yaml.jsx

```diff
@@ -87,6 +90,7 @@ export const EditYAML = connect(stateToProps)(
         stale: false,
         sampleObj: props.sampleObj,
         fileUpload: props.fileUpload,
+        showSidebar: (props.create && props.create) || false,
```

Comment:
Is it intentional to hide the sidebar by default when editing?

Reply:
That was how it was set up originally on master; I tried to keep the behavior the same.

Reply:
Isn't this just....

```suggestion
        showSidebar: props.create,
```